src/test/java/uk/gov/companieshouse/items/orders/api/ItemTypeUnitTest.java

```diff
@@ -21,64 +20,64 @@
     void certificateItemPopulatedCorrectly() {
 
         // Given
-        final CertificateItemDTO certificateItemDTO = new CertificateItemDTO();
+        final Item item = new Item();
 
         // When
-        CERTIFICATE.populateReadOnlyFields(certificateItemDTO);
+        CERTIFICATE.populateReadOnlyFields(item);
 
         // Then
-        verifyDescriptionFields(certificateItemDTO, "certificate");
-        verifyCostsFields(certificateItemDTO);
-        verifyPostalDelivery(certificateItemDTO, CERTIFICATE);
+        verifyDescriptionFields(item, "certificate");
+        verifyCostsFields(item);
+        verifyPostalDelivery(item, CERTIFICATE);
     }
 
     @Test
     @DisplayName("Certified copy item populated correctly")
     void certifiedCopyItemPopulatedCorrectly() {
 
         // Given
-        final CertificateItemDTO certificateItemDTO = new CertificateItemDTO();
+        final Item item = new Item();
 
         // When
-        CERTIFIED_COPY.populateReadOnlyFields(certificateItemDTO);
+        CERTIFIED_COPY.populateReadOnlyFields(item);
 
         // Then
-        verifyDescriptionFields(certificateItemDTO, "certified-copy");
-        verifyCostsFields(certificateItemDTO);
-        verifyPostalDelivery(certificateItemDTO, CERTIFIED_COPY);
+        verifyDescriptionFields(item, "certified-copy");
+        verifyCostsFields(item);
+        verifyPostalDelivery(item, CERTIFIED_COPY);
     }
 
     @Test
     @DisplayName("Scan on demand item populated correctly")
     void scanOnDemandItemPopulatedCorrectly() {
 
         // Given
-        final CertificateItemDTO certificateItemDTO = new CertificateItemDTO();
+        final Item item = new Item();
 
         // When
-        SCAN_ON_DEMAND.populateReadOnlyFields(certificateItemDTO);
+        SCAN_ON_DEMAND.populateReadOnlyFields(item);
 
         // Then
-        verifyDescriptionFields(certificateItemDTO, "scan-on-demand");
-        verifyCostsFields(certificateItemDTO);
-        verifyPostalDelivery(certificateItemDTO, SCAN_ON_DEMAND);
+        verifyDescriptionFields(item, "scan-on-demand");
+        verifyCostsFields(item);
+        verifyPostalDelivery(item, SCAN_ON_DEMAND);
```

Comment:
The previous 3 tests are duplicated with a different certificate type. Can they be refactored to remove the duplication please?

Reply:
Yes, good call. Will do.